server/scripts/participant-stats-rejected.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const csv = require('fast-csv');
+const path = require('path');
+const { writeFileSync } = require('fs');
+const { dbClient } = require('../db/db.js');
+const { getRejectedParticipantsReport } = require('../services/reporting');
+
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+
+      const results = await getRejectedParticipantsReport();
+
+      console.log(results);
+
+      const csvStream = csv.format({ headers: true });
+
+      results.forEach((result) => {
+        csvStream.write({
+          'Participant Name': `${result.participantInfo.firstName} ${result.participantInfo.lastName}`,
+          'Participant ID': result.participantId,
+          'Employer ID': result.employerId,
+          'Reason for Rejection': result.rejection.final_status,
```

Comment:
I realize the story didn't say which fields to capture, sorry - it was definitely written assuming we would expand on the in-progress report

I think the fields we should have are:

- Participant ID
- Employer ID
- Employer email
- Employer health region(s)
- Reason for Rejection
- Date*

*date of the rejection action being taken

I think these are all in the other report(s) so hopefully easy to add :)